src/js/_enqueues/admin/site-icon.js

```diff
@@ -1,15 +1,40 @@
-(function($) {
```

Comment:
OK, turns out I'm not the first one to notice this.

There was https://core.trac.wordpress.org/ticket/43135 and https://core.trac.wordpress.org/ticket/28236, where https://core.trac.wordpress.org/ticket/42850 only removed the version used in `script-loader.php` but not the dependency.

And ESLint hasn't been implemented yet, see https://core.trac.wordpress.org/ticket/31823.

Since https://core.trac.wordpress.org/ticket/28236 is still open, the best course of action seems to be:

1. Apply patch from [#28236](https://core.trac.wordpress.org/ticket/28236) to remove outdated configs
2. Use `"esversion": 6`
3. Close that ticket and focus on ESLint next

I'll self-assign that ticket to get that done.

Reply:
Until https://core.trac.wordpress.org/ticket/28236 has a fix in trunk, I don't want to force a failing build. Unless you would consider this blocked by that, I'm going to wait to resolve this.

Reply:
Yeah no waiting is fine, that‘s what I meant 👍

Reply:
This is now committed in https://core.trac.wordpress.org/changeset/57702

Reply:
thank you! I merged trunk and reupdated the code.